cardano-cli/src/Cardano/CLI/Compatible/Run.hs

```diff
@@ -32,6 +33,15 @@ runAnyCompatibleCommand (AnyCompatibleCommand cmd) = runCompatibleCommand cmd
 
 runCompatibleCommand :: CompatibleCommand era -> ExceptT CompatibleCmdError IO ()
 runCompatibleCommand (CompatibleTransactionCmd txCmd) =
-  firstExceptT CompatibleTransactionError $ runCompatibleTransactionCmd txCmd
+  liftIO $
+    executeRio
+      (runCompatibleTransactionCmd txCmd)
 runCompatibleCommand (CompatibleGovernanceCmds govCmd) =
   firstExceptT CompatibleGovernanceError $ runCompatibleGovernanceCmds govCmd
+
+-- | This is a temporary function until all commands are migrated to RIO
+-- Once this happens we can remove this function and rely on `toplevelExceptionHandler`
+executeRio :: RIO () () -> IO ()
```

Comment:
What about an alias `type CIO e a = HasCallStack => RIO e a` ?

Reply:
Yeah that would make it easier to enforce `HasCallStack`  :+1: